addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1781,31 +1809,54 @@ const ListUserValueWidget = UserValueWidget.extend({
         if (this.addItemButton) {
             this.addItemButton.remove();
         }
-        if (this.isCustom) {
-            this.addItemButton = document.createElement('we-button');
-            this.addItemButton.textContent = this.addItemTitle;
-            this.addItemButton.classList.add('o_we_list_add_optional');
-            currentValues.forEach(el => this._addItemToTable(el, el));
+
+        if (this.createWidget) {
+            const selectedIds = currentValues.map(({ id }) => id)
```

Comment:
I'd remove the spaces around `id` :grimacing:

Reply:
Seems like everyone else in odoo has started using `object-curly-spacing` so I thought I'd align my style with the other teams. I don't really have a preference either direction but I think readability benefits from the consistency. Do you think we should keep no spacing in the website modules?

Reply:
I think everyone else is simply not caring about any rule anymore :wink: Let's not care either, I tried, I am tired :smile: :shrug: